
import io;

function int foo(int x)
{
  return 5;
} (10);
// ^ should be an error

